### PR TITLE
Hw1

### PR DESCRIPTION
**Pull request by heeveG**

### CODE REVIEW

**Comment by heeveG** [comment]
https://confident-mclean-7328e1.netlify.app/

File: src/components/header.js [hunk 1]
```diff
@@ -5,7 +5,11 @@ import "../styles/header.css"
 function Header() {
     return (
         <div className="header">
-            Header
+            <ul className="header-ul">
```
Comment (pavlo-berezin) [inline src/components/header.js:8]: You already know that this is `ul` from the tag name, no need to specify it in the class name.

File: src/styles/header.css [hunk 2]
```diff
@@ -2,4 +2,17 @@
...
+    color: grey;
+}
+
+.header-ul{
+    list-style-type: none;
+    padding: 1px;
+}
+
+.header-li {
+    float: left;
```
Comment (pavlo-berezin) [inline src/styles/header.css:14]: floats are kinda meh at this point, consider using something else.

File: src/styles/subheader.css [hunk 3]
```diff
@@ -1,6 +1,17 @@
...
+}
+
+.subheader-ul{
+    list-style-type: none;
+    padding: 15px;
+}
+
+.subheader-li {
+    float: left;
+    padding-right:  5%;
```
Comment (pavlo-berezin) [inline src/styles/subheader.css:16]: extra space.

**Pull request merged**